app/src/main/java/com/safetyheads/akademiaandroida/network/ApiClient.kt

```diff
@@ -35,4 +38,17 @@ class ApiClient {
         return this
     }
 
+    private fun OkHttpClient.Builder.addCustomInterceptor(): OkHttpClient.Builder {
+        addInterceptor { chain ->
```

Comment:
`addInterceptor(HttpLoggingInterceptor().setLevel(HttpLoggingInterceptor.Level.NONE))`
Should be enough.